state[16] dq
float64[] dqdot
state[16] dqddot

state[16] Kp
state[16] Kd
int32 SubIndex
